docs/source/get_started/examples.rst

```diff
@@ -1,7 +1,16 @@
 Examples
```

Comment:
The main modification is to fix the cross-reference links like `load_sampl` or `MultitaskRegressor`